addons/calendar/models/calendar_event.py

```diff
@@ -1613,6 +1633,38 @@ def ics_datetime(idate, allday=False):
 
         return result
 
+    @api.model
+    def _get_contact_details_description(self, organizer, partners):
+        """Build sanitized HTML with the organizer details and the details
+        of the contact partner (the first partner which is not the organizer).
+        """
+        odoobot = self.env.ref('base.user_root')
+        contact_description = []
+        # Organizer
+        if organizer and organizer.id != odoobot.id:
+            contact_description.extend(self._prepare_partner_details_html(_("Organized by"), organizer.partner_id))
+        # First contact partner
+        first_partner = partners.filtered(lambda partner: (
+            partner != odoobot.partner_id and
+            (not organizer or partner != organizer.partner_id)
```

Comment:
If we have no organizer, organize.partner_id is just a void recordset, hence it should be equivalent to
```suggestion
        first_partner = partners.filtered(lambda partner: partner not in (odoobot.partner_id+ organizer.partner_id))
```